eclass/toolchain.eclass

```diff
@@ -84,6 +84,11 @@ tc_version_is_between() {
 # Used to override GCC version. Useful for e.g. live ebuilds or snapshots.
 # Defaults to ${PV}.
 
+# @ECLASS_VARIABLE: TOOLCHAIN_GCC_VALIDATE_FAILURES_VERSION
+# @DESCRIPTION:
+# Version of test comparison script (validate_fails.py) to use.
+: ${GCC_VALIDATE_FAILURES_VERSION:=7bbfb01a32b73842f8908de028703510a0e12057}
```

Comment:
In case anyone missed what this was about in gnetoo-qa, it's to avoid DoS with globs given it'll resolve them and crawl the entire filesystem. And QA decided that to do this everywhere rather than nitpick which aren't needed.

It's also a shellcheck warning, see the last point in https://www.shellcheck.net/wiki/SC2086